kubernetes/usage.md

```
@@ -0,0 +1,251 @@
+### Azure
+
+If one was using Azure, that might be accomplished with 
+```
+$ az network dns record-set a add-record -g myresourcegroup -z example.com -a 75.72.233.202 -n journiv
+```
+
+### GCP 
+
+Google's CloudDNS might use
+```
+$ gcloud dns record-sets update journivv --rrdatas=75.72.233.202 --type=A --ttl=300 --zone="mydnszone" --project=mygcpproject
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)